lms/urls.py

```diff
@@ -237,6 +237,19 @@
     )
 
 if settings.COURSEWARE_ENABLED:
+    COURSE_URLS = patterns(
```

Comment:
I can't use the words TODO in the code, because it'll trigger a pylint violation (seriously). Filed: https://openedx.atlassian.net/browse/SOL-956